### PR TITLE
sets up Sentry for crash reporting

### PR DESCRIPTION
**Pull request by alexosugo**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -361,6 +361,6 @@
 	<string name="language">Language</string>
 	<string name="bounty">Bounty</string>
 
-
+	<string name="sentry_dsn">https://ce9bf386ab104ece903cd33fc563c241@o44209.ingest.sentry.io/5477330</string>
```
Comment (davkutalek) [inline app/src/main/res/values/strings.xml:364]: Put this at the top of the file and don't forget `translatable="false"`

File: app/src/main/java/com/hover/stax/ApplicationInstance.java [hunk 2]
```diff
@@ -45,6 +51,29 @@ private void setLogger() {
             Timber.uprootAll();
         }
     }
+
+    private void initSentry(){
```
Comment (davkutalek) [inline app/src/main/java/com/hover/stax/ApplicationInstance.java:55]: Do we have to use manual initialization for Stax?

Comment (alexosugo) [reply app/src/main/java/com/hover/stax/ApplicationInstance.java:55]: From my tests, it seems the `SDK` configurations override Stax's and the events aren't recorded.

Comment (davkutalek) [reply app/src/main/java/com/hover/stax/ApplicationInstance.java:55]: Cool, was just curious. It does work this way?

**Pull request closed without merging**